docs/examples/sketch_000/sketch_000.py

```diff
@@ -3,11 +3,9 @@
 
 def setup():
     createCanvas(200, 200)
-    background(160)
-
 
 def draw():
-    fill("blue")
     background(200)
-    radius = sin(frameCount / 60) * 50 + 50
-    ellipse(100, 100, radius, radius)
+    diameter = sin(frameCount / 60) * 50 + 50
+    fill("blue")
+    ellipse(100, 100, diameter, diameter)
```

Comment:
Shouldn't we replace this `ellipse` call by a `circle` one instead?

Reply:
hmm, yes, we could, but I guess ellipse is alright.